server/sonar-db-core/src/main/resources/org/sonar/db/version/schema-h2.ddl

```diff
@@ -561,6 +561,8 @@ CREATE TABLE "CE_QUEUE" (
   "COMPONENT_UUID" VARCHAR(40) NULL,
   "STATUS" VARCHAR(15) NOT NULL,
   "SUBMITTER_LOGIN" VARCHAR(255) NULL,
+  "WORKER_UUID" VARCHAR(40) NULL,
```

Comment:
To be verified, but I think that the table `CE_QUEUE` is truncated on version upgrades. In that case column could be NOT NULL.